src/hexo/themes/documentation/source/images/results-warning-icon.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 13.2 11.2"><style>.st0{fill:#FFFFFF;} .st1{fill:none;stroke:#EA664F;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;} .st2{opacity:0.9;fill-rule:evenodd;clip-rule:evenodd;fill:#101072;} .st3{fill:url(#background_5_);} .st4{fill:none;} .st5{fill:#4D4D4D;} .st6{fill:#4046DD;} .st7{fill:#2D28A1;} .st8{fill:none;stroke:#333333;stroke-miterlimit:10;} .st9{fill:#F2F2F2;} .st10{fill:#666666;} .st11{fill:#CCCCCC;} .st12{fill:#EA5238;} .st13{fill:#333333;} .st14{fill:none;stroke:#E2E2E2;stroke-miterlimit:10;} .st15{fill:#1A1A1A;} .st16{fill:none;stroke:#4046DD;stroke-width:3;stroke-miterlimit:10;} .st17{fill:#292F56;} .st18{fill-rule:evenodd;clip-rule:evenodd;fill:#4046DD;} .st19{fill-rule:evenodd;clip-rule:evenodd;fill:none;} .st20{fill:none;stroke:#051917;stroke-miterlimit:10;} .st21{fill:#4D4D4D;stroke:#FFFFFF;stroke-width:2;stroke-miterlimit:10;} .st22{fill:#636464;stroke:#FFFFFF;stroke-miterlimit:10;} .st23{fill:#636464;} .st24{fill:#9C9EA8;} .st25{fill:#F2D5CB;} .st26{fill:#F4F4F4;} .st27{fill:#D8D9F7;} .st28{fill:#B91B06;} .st29{fill:none;stroke:#000000;stroke-width:10;stroke-miterlimit:10;} .st30{fill:#AFAFAF;} .st31{fill:#E6E6E6;} .st32{fill:#B3B3B3;} .st33{fill:#ACADDA;} .st34{fill:#D33E2A;} .st35{fill:#817EC7;} .st36{fill:#5753B4;} .st37{fill:#0B0687;} .st38{fill:url(#background_6_);} .st39{fill:#D43E2B;} .st40{fill:#6E33B4;} .st41{fill:none;stroke:#CCCCCC;stroke-miterlimit:10;} .st42{fill:none;stroke:#000000;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;} .st43{fill:url(#background_7_);} .st44{opacity:0.12;fill:#EA664F;} .st45{opacity:0.85;fill:#5753B4;} .st46{opacity:0.85;fill:#817EC7;} .st47{opacity:0.53;fill:#0B0687;} .st48{fill:url(#background_8_);} .st49{fill:none;stroke:#4046DD;stroke-width:4;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;} .st50{fill:#6125C3;} .st51{fill:#2A2E90;} .st52{fill:#E5DDF8;} .st53{fill:#F37A1A;} .st54{fill:#80EDC5;} .st55{fill:#66559D;} .st56{fill:#EA664F;} .st57{fill:#5065FF;} .st58{fill:#00007F;} .st59{fill:#E5745C;} .st60{fill:#2E9E88;} .st61{fill:#00479E;} .st62{fill:#5F00EB;} .st63{fill:#002452;}</style><g id="Layer_1"><path class="st12" d="M12.3 10.8H.9c-.2 0-.3-.1-.4-.2s-.1-.3 0-.4L6.2.6c.1-.1.2-.2.4-.2s.3.1.4.2l5.7 9.6c.1.1.1.3 0 .4-.1.2-.2.2-.4.2zM1.7 10h9.9l-5-8.3L1.7 10z"/><path class="st12" d="M6.9 7.6L7 4.1h-.8l.1 3.5zM6.6 8c-.3 0-.5.3-.5.5 0 .3.2.5.5.5s.5-.2.5-.5c0-.2-.2-.5-.5-.5z"/></g></svg>
```

Comment:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 13.2 11.2"><style>.e{fill:#ea5238}</style><path class="e" d="M12.3 10.8H.9c-.2 0-.3-.1-.4-.2s-.1-.3 0-.4L6.2.6c.1-.1.2-.2.4-.2s.3.1.4.2l5.7 9.6c.1.1.1.3 0 .4-.1.2-.2.2-.4.2zM1.7 10h9.9l-5-8.3L1.7 10z"/><path class="e" d="M6.9 7.6L7 4.1h-.8l.1 3.5zM6.6 8c-.3 0-.5.3-.5.5 0 .3.2.5.5.5s.5-.2.5-.5c0-.2-.2-.5-.5-.5z"/></svg>
```

Reply:
@qzhou1607 can you update this file with the content of @alrra's comment?